frontend/apps/reader/modules/readergesture.lua

```diff
@@ -631,6 +631,36 @@ Default value: %1]]), GestureDetector.PAN_DELAYED_INTERVAL/1000),
                     UIManager:show(items)
                 end,
             },
+            {
+                text = _("Hold pan rate"),
+                callback = function()
+                    local SpinWidget = require("ui/widget/spinwidget")
+                    local current_value = G_reader_settings:readSetting("pan_rate")
+                    if not current_value then
+                        current_value = Screen.low_pan_rate and 2.0 or 30.0
+                    end
+                    local items = SpinWidget:new{
+                        text = T(_([[
+Used when highlighting text, bigger value faster refresh and bigger CPU usage.
+5.0 should be good for most e-ink ereaders.
+Default value: %1]]), current_value),
+                        width = math.floor(Screen:getWidth() * 0.6),
+                        value = current_value,
+                        value_min = 1.0,
+                        value_max = 30.0,
```

Comment:
Or 60? Not for E Ink of course but on desktop or a regular Android device.

Reply:
Sure, I can change that, but tbh I don't see a point. 30 gives 1 refresh every ~33ms. I'd say 30 and 60 don't have noticeable difference.

Reply:
Matches the non-gaming standard of 60Hz monitors ;).

Reply:
Perfectly fine as a default, annoying as an artificial limitation. ;-)

Reply:
Oh, right, didn't notice this was the max, not the default ;).